parlai/utils/torch.py

```diff
@@ -226,6 +226,29 @@ def argsort(keys: List[Any], *lists: List[List[Any]], descending: bool = False):
     return output
 
 
+def compute_grad_norm(parameters, norm_type=2.0):
+    """
+    Compute norm over gradients of model parameters.
```

Comment:
It wants an extra newline between this and the next line, and it wants you to delete the extra empty line at the end.